src/runner.py

```diff
@@ -0,0 +1,50 @@
+"""
+Оформите решение задач из прошлых домашних работ в функции.
+Напишите функцию runner.
+runner() – все фукнции вызываются по очереди
+runner(‘func_name’) – вызывается только функцию func_name.
+runner(‘func’, ‘func1’...) - вызывает все переданные функции
+"""
+import string
+
+
+def func_1():
+    null_str = ''
+    input_line = 'abc cde def'
+
+    for p in input_line.split():
+        f = p.strip(string.punctuation)
+        for el in f:
+            if el not in null_str:
+                null_str += el
+
+    print(null_str)
+
+
+def func_2():
+    lst_gen = [a + b for a in 'abbcc' for b in ['a', 'b', 'c']]
+    for el in lst_gen:
+        if lst_gen.count(el) == 1:
+            print(el)
+
+
+def func_3():
+    lst_ = [1, 2, 0, 5, 0, 7, 8, 9, 0]
+    print([el for el in lst_ if el] + [0] * lst_.count(0))
+
+
+def runner(*args):
```

Comment:
you can use getattr and check if smth is callable and it would grant you possibility to run any functions from imported modules

Reply:
it works only for this functions and not flexible